### ISSUE TITLE
(Accessibility) Yellow-on-white icon for KeyVaults is hard to see

### ISSUE THREAD

**Issue by bwateratmsft** [microsoft/vscode-azureresourcegroups#173]
The yellow-on-white icon for KeyVaults is not easy to see clearly:

![image](https://user-images.githubusercontent.com/36966225/163433002-f0ce6dbb-bd7b-49a5-ac6c-103415cf4e10.png)

**Comment by nturinski** [comment]
Should we actually support having a light/dark theme icon for each Azure Resource? Currently we just have one icon per resource.

**Comment by bwateratmsft** [comment]
This is on the dark theme--the icon contains a white circle. It'd look the same on light theme.

**Comment by nturinski** [comment]
Yeah, I meant like how a lot of our extensions have a light/dark resource folder such as Functions https://github.com/microsoft/vscode-azurefunctions/tree/main/resources. Would definitely be better to just have one that looks good on both though.
